### PR TITLE
Downgrade to AspNetCore 2.1 for better compatibility with deployed en…

### PR DESCRIPTION
**Pull request by martinsmith1968**
…vironments

### CODE REVIEW

File: ClearBank.Signing.Api/ClearBank.Signing.Api/ClearBank.Signing.Api.csproj [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
```
Comment (IvanAlekseev) [inline ClearBank.Signing.Api/ClearBank.Signing.Api/ClearBank.Signing.Api.csproj]: 2.1 then right?

File: ClearBank.Signing.Api/ClearBank.Signing.Api/Controllers/SigningController.cs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.IO;
+using System.Reflection;
```
Comment (IvanAlekseev) [inline ClearBank.Signing.Api/ClearBank.Signing.Api/Controllers/SigningController.cs:3]: WHAT? Why?

File: ClearBank.Signing.Api/ClearBank.Signing.Api/PlainTextInputFormatter.cs [hunk 3]
```diff
@@ -23,17 +23,19 @@ public override async Task<InputFormatterResult> ReadRequestBodyAsync(InputForma
             if (context == null) throw new ArgumentNullException(nameof(context));
             if (effectiveEncoding == null) throw new ArgumentNullException(nameof(effectiveEncoding));
 
-            using var reader = new StreamReader(context.HttpContext.Request.Body, effectiveEncoding);
-            try
+            using (var reader = new StreamReader(context.HttpContext.Request.Body, effectiveEncoding))
```
Comment (IvanAlekseev) [inline ClearBank.Signing.Api/ClearBank.Signing.Api/PlainTextInputFormatter.cs:26]: So ugly, so old school, so last century

File: ClearBank.Signing.Api/ClearBank.Signing.Api/ClearBank.Signing.Api.csproj (hunk 1 above)
Comment (martinsmith1968) [reply ClearBank.Signing.Api/ClearBank.Signing.Api/ClearBank.Signing.Api.csproj]: Yeah. Nice - thanks Visual Studio wizard

File: ClearBank.Signing.Api/ClearBank.Signing.Api/Controllers/SigningController.cs (hunk 2 above)
Comment (martinsmith1968) [reply ClearBank.Signing.Api/ClearBank.Signing.Api/Controllers/SigningController.cs:3]: Needed by Assembly

File: ClearBank.Signing.Api/ClearBank.Signing.Api/PlainTextInputFormatter.cs (hunk 3 above)
Comment (martinsmith1968) [reply ClearBank.Signing.Api/ClearBank.Signing.Api/PlainTextInputFormatter.cs:26]: So necessary

**Review by IvanAlekseev — APPROVED** [review]

**Pull request merged**